luminance-sdl2/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "luminance-sdl2"
+version = "0.1.0"
+license = "BSD-3-Clause"
+authors = [
+  "Dimitri Sabadie <dimitri.sabadie@gmail.com>",
+  "Joel Nordström <e.joel.nordstrom@gmail.com>"
+]
+description = "Sdl2 support for luminance"
```

Comment:
Can we use `SDL2` is they’re using it in the `sdl2` crate, please? :)

Reply:
Absolutely!